src/test/java/com/exasol/releaserobot/repository/maven/JavaMavenGitBranchContentTest.java

```diff
@@ -21,8 +22,8 @@
 import org.mockito.Mock;
 import org.mockito.junit.jupiter.MockitoExtension;
 
-import com.exasol.releaserobot.repository.Branch;
 import com.exasol.releaserobot.repository.GitRepositoryException;
+import com.exasol.releaserobot.usecases.Repository;
 
 @ExtendWith(MockitoExtension.class)
 class JavaMavenGitBranchContentTest {
```

Comment:
the test name does not match with the class now, right?

Reply:
fixed it